src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -98,7 +98,7 @@ public static Deck parseDeck(String deckName) throws ParseException {
         requireNonNull(deckName);
         String trimmedDeckName = deckName.trim();
         if (!Answer.isValidAnswer(trimmedDeckName)) {
```

Comment:
we should create a DeckName class soon. Right now we are still checking validAnswer on deck names...

Reply:
#148